src/OrchardCore.Cms.Web/appsettings.json

```diff
@@ -216,8 +216,8 @@
     //  "CallbackPath": "/signin-oidc",
     //  "SaveTokens": false
     //},
-    //"OrchardCore_Tenants": {
-    //  "TenantRemovalAllowed": true // Whether tenant removal is allowed, false by default.
-    //}
+    "OrchardCore_Tenants": {
+      "TenantRemovalAllowed": true // Whether tenant removal is allowed, false by default.
```

Comment:
If it's `false` by default how you enable it in `appsettings.json`?

Reply:
Oops should be commented out, I uncommented it only for testing, thanks!

Here by default means that if you don't provide a config value it is set to false.

Otherwise it is set to the value you explicitly provide here.